config/software/metasploit-framework.rb

```diff
@@ -37,6 +37,7 @@
 
 # Also whitelist sqlite deps too as libz is provided just not first on path
 whitelist_file "#{install_dir}/embedded/lib/ruby/gems/#{ruby_abi_version}/gems/sqlite3-.*"
+whitelist_file "#{install_dir}//embedded/lib/ruby/gems/#{ruby_abi_version}/.*/sqlite3.*"
```

Comment:
Just confirming if this should be:

```suggestion
whitelist_file "#{install_dir}/embedded/lib/ruby/gems/#{ruby_abi_version}/.*/sqlite3.*"
```

Reply:
Yes the `//` is required, the match fails to allow the files otherwise.

Reply:
Is there any context why? 👀 

I would've assumed both of the whitelist_file call would have the same search prefix